butterknife-compiler/src/main/java/butterknife/compiler/ButterKnifeProcessor.java

```diff
@@ -666,9 +669,10 @@ private void parseResourceColor(Element element,
     int id = element.getAnnotation(BindColor.class).value();
     Id resourceId = elementToId(element, BindColor.class, id);
     BindingSet.Builder builder = getOrCreateBindingBuilder(builderMap, enclosingElement);
-    builder.addResource(new FieldResourceBinding(resourceId, name,
-        isColorStateList ? FieldResourceBinding.Type.COLOR_STATE_LIST
-            : FieldResourceBinding.Type.COLOR));
+    builder.addResource(new FieldResourceBinding(resourceId, name, isColorStateList ? (useAndroidX
+        ? FieldResourceBinding.Type.COLOR_STATE_LIST_ANDROIDX
+        : FieldResourceBinding.Type.COLOR_STATE_LIST) : (useAndroidX
+        ? FieldResourceBinding.Type.COLOR_ANDROIDX : FieldResourceBinding.Type.COLOR)));
```

Comment:
Double nested ternary is hurting my brain. Can we pull each branch of the outermost ternary into a local so that the result is just `isColorStateList ? colorStateListType : colorType`?

Reply:
24afa90